docs/using/api.ipynb

```diff
@@ -2,156 +2,155 @@
     ":width: 500 px\n",
     "\n",
     "Illustration of the execution process.\n",
     "```"
-   ]
+   ],
+   "metadata": {}
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
     "```{note}\n",
-    "The full Jupyter notebook for this page can accessed here; {nb-download:notebook.ipynb}`api`.\n",
+    "The full Jupyter notebook for this page can accessed here; {nb-download}`api.ipynb`.\n",
     "Try it for yourself!\n",
     "```"
-   ]
+   ],
+   "metadata": {}
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
     "## Initialisation"
-   ]
+   ],
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": 1,
-   "metadata": {},
-   "outputs": [],
    "source": [
     "from pathlib import Path\n",
     "import nbformat as nbf\n",
     "from jupyter_cache import get_cache\n",
-    "from jupyter_cache.base import NbBundleIn\n",
+    "from jupyter_cache.base import CacheBundleIn\n",
     "from jupyter_cache.executors import load_executor, list_executors\n",
     "from jupyter_cache.utils import (\n",
     "    tabulate_cache_records, \n",
-    "    tabulate_stage_records\n",
+    "    tabulate_project_records\n",
     ")"
-   ]
+   ],
+   "outputs": [],
+   "metadata": {}
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
     "First we setup a cache and ensure that it is cleared.\n",
     "\n",
     "```{important}\n",
     "Clearing a cache wipes its entire content, including any settings (such as cache limit).\n",
     "```"
-   ]
+   ],
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": 2,
-   "metadata": {},
+   "source": [
+    "cache = get_cache(\".jupyter_cache\")\n",
+    "cache.clear_cache()\n",
+    "cache"
+   ],
    "outputs": [
     {
+     "output_type": "execute_result",
      "data": {
       "text/plain": [
-       "JupyterCacheBase('/Users/cjs14/GitHub/jupyter-cache/docs/using/.jupyter_cache')"
+       "JupyterCacheBase('/Users/chrisjsewell/Documents/GitHub/jupyter-cache/docs/using/.jupyter_cache')"
       ]
      },
-     "execution_count": 2,
      "metadata": {},
-     "output_type": "execute_result"
+     "execution_count": 2
     }
    ],
-   "source": [
-    "cache = get_cache(\".jupyter_cache\")\n",
-    "cache.clear_cache()\n",
-    "cache"
-   ]
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": 3,
-   "metadata": {},
+   "source": [
+    "print(cache.list_cache_records())\n",
+    "print(cache.list_project_records())"
+   ],
    "outputs": [
     {
-     "name": "stdout",
      "output_type": "stream",
+     "name": "stdout",
      "text": [
       "[]\n",
       "[]\n"
      ]
     }
    ],
-   "source": [
-    "print(cache.list_cache_records())\n",
-    "print(cache.list_staged_records())"
-   ]
+   "metadata": {}
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
     "(use/api/cache)=\n",
     "\n",
     "## Cacheing Notebooks"
```

Comment:
I think Cacheing Notebooks should come after the staging/execution examples, since that would mirror the same structure that the `Command-Line` page uses. And since staging/execution is more common than Cacheing, presumably?